src/rulesets/oas2/index.json

```diff
@@ -179,6 +182,16 @@
       "tags": [
         "schema"
       ]
+    },
+    "oas2-unused-definition": {
+      "description": "Unused definition has been detected.",
+      "recommended": true,
```

Comment:
@philsturgeon Is it ok to tag this as recommended?

Reply:
If this is a warning then yeah a recommended is fine with me.